source/cloud-security/github/monitoring-github-activity.rst

```diff
@@ -138,32 +138,4 @@ If Wazuh successfully connects to GitHub, the events raised above will trigger t
         },
         "location":"github"
     }
-    
-
-Enabling dashboard visualization  
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-You can activate the corresponding Security Information Management module on the Wazuh Dashboard. This module provides additional details and insights about events, as shown in the screenshots below.
-
-    .. thumbnail:: /images/github/github-dashboard.png
-       :title: GitHub dashboard
-       :alt: GitHub dashboard
-       :align: center
-       :width: 80%
-
-    .. thumbnail:: /images/github/github-events.png
-       :title: GitHub events
-       :alt: GitHub events
-       :align: center
-       :width: 80%
```

Comment:
**thought**: this could require rephrasing because the visibility of the module is not configurable. If this was removed, the images could be removed too.

Reply:
Ok, I will remove the related images too.

Reply:
Fixed in ef02e7cd655d1412e5f49256400d4ba4196357cc.